src/test/java/org/opentripplanner/generate/doc/framework/DocsTestConstants.java

```diff
@@ -24,7 +24,7 @@ static boolean docsExistOrWarn() {
     }
     LOG.warn(
       """
-      SKIP TEST - '/docs' NOT FOUND
+      SKIP TEST - '/doc' NOT FOUND
```

Comment:
`/doc/user` here as well

Reply:
Yeah you're right. But what about the warn string just below in the LOG.warn? 

`The doc/templates directory might not be available if you run the tests outside the
root of the projects. This may happen if the project root is not the working directory,
if you run tests using jar files or in a Maven multi-module project.`

Shouldn't we check also if the doc/template directory exist? And change the comment accordingly quoting both doc/user and doc/templates.

Reply:
Yes sure, I think that solution would be the most correct one.